packages/@uppy/dashboard/src/components/AddFiles.js

```diff
@@ -1,101 +1,113 @@
+    return <a tabindex="-1" href="https://uppy.io" rel="noreferrer noopener" target="_blank" class="uppy-Dashboard-poweredBy">
+      {this.props.i18n('poweredBy')}
+      <svg aria-hidden="true" class="UppyIcon uppy-Dashboard-poweredByIcon" width="11" height="11" viewBox="0 0 11 11">
+        <path d="M7.365 10.5l-.01-4.045h2.612L5.5.806l-4.467 5.65h2.604l.01 4.044h3.718z" fill-rule="evenodd" />
+      </svg>
+      <span class="uppy-Dashboard-poweredByUppy">Uppy</span>
+    </a>
+  }
 
-    if (!hasAcquirers) {
-      return (
-        <div class="uppy-DashboardAddFiles">
-          <div class="uppy-DashboardTabs">
-            <ActionBrowseTagline
-              acquirers={this.props.acquirers}
-              handleInputChange={this.props.handleInputChange}
-              i18n={this.props.i18n}
-              i18nArray={this.props.i18nArray}
-              allowedFileTypes={this.props.allowedFileTypes}
-              maxNumberOfFiles={this.props.maxNumberOfFiles}
-            />
-          </div>
-          <div class="uppy-DashboardAddFiles-info">
-            { this.props.note && <div class="uppy-Dashboard-note">{this.props.note}</div> }
-            { this.props.proudlyDisplayPoweredByUppy && poweredByUppy(this.props) }
-          </div>
-        </div>
-      )
-    }
+  renderHiddenFileInput () {
+    return <input class="uppy-Dashboard-input"
+      hidden
+      aria-hidden="true"
+      tabindex={-1}
+      type="file"
+      name="files[]"
+      multiple={this.props.maxNumberOfFiles !== 1}
+      // Clear file input
+      onchange={(event) => {
+        this.props.handleInputChange(event)
+        event.target.value = null
+      }}
+      accept={this.props.allowedFileTypes}
+      ref={(ref) => { this.fileInput = ref }} />
+  }
+
+  renderDropPasteBrowseTagline () {
+    const browse =
+      <button type="button"
+        class="uppy-Dashboard-browse"
+        onclick={this.triggerFileInputClick}>
+        {this.props.i18n('browse')}
+      </button>
 
-    // empty value="" on file input, so that the input is cleared after a file is selected,
-    // because Uppy will be handling the upload and so we can select same file
-    // after removing — otherwise browser thinks it’s already selected
+    return <div class="uppy-Dashboard-dropFilesTitle">
+      {this.props.acquirers.length === 0
+        ? this.props.i18nArray('dropPaste', { browse })
+        : this.props.i18nArray('dropPasteImport', { browse })
+      }
+    </div>
+  }
+
+  renderMyDeviceAcquirer () {
+    return <div class="uppy-DashboardTab" role="presentation">
+      <button type="button"
+        class="uppy-DashboardTab-btn"
+        role="tab"
+        tabindex={0}
+        onclick={this.triggerFileInputClick}>
+        {localIcon()}
+        <div class="uppy-DashboardTab-name">{this.props.i18n('myDevice')}</div>
+      </button>
+    </div>
+  }
+
+  renderAcquirer (acquirer) {
+    return <div class="uppy-DashboardTab" role="presentation">
```

Comment:
Nitpick: since you’ve refactored this (thanks!), could we add `()` after all of the `return` statements, maybe?

Reply:
@arturi, let's, but let's add eslint rules for these cases?
I also see both lowercase props (e.g. onpaste) and camelCase (onPaste) used interchangeably, that's something that's nice to have handled by eslint.

Reply:
Yes, good idea about eslint for those. But maybe in a separate PR after accessibility and Alex’s PRs are merged? Cause we’ll need to touch everything with strict rules.

Reply:
Agreed.